specs/plugin-extensibility.md

```diff
@@ -22,13 +22,28 @@ Keys and associated certificates used for signing artifacts using Notation could
 
 Notation will invoke plugins as executable, pass parameters using command line arguments, and use standard IO streams to pass request/response payloads. This mechanism is used as Go language (used to develop [Notation Go library](https://github.com/notaryproject/notation-go)) does not have a [in built support](https://github.com/golang/go/issues/19282) to load and execute plugins that works across OS platforms. Other mechanisms like gRPC require every plugin to be implemented as a service/daemon.
 
+### Package and distribute plugin file
+
+To ensure plugin installation is compatible with Notation, plugin publisher SHOULD follow these conventions to package and distribute plugin files:
+
+* `.zip`, `.tar.gz`, and single plugin executable file formats are supported by Notation. Plugin executable file MUST follow the naming convention `notation-{plugin-name}` 
+* Plugin publisher SHOULD distribute the plugin file with an `https` URL to enable install a plugin from a remote source, or provide an archive file to enable install a plugin from filesystem
+* If multiple files are compressed in the `.zip` or `.tar.gz` file, there MUST be only one plugin executable file exists
+* Plugin publisher SHOULD provide SHA256 checksum for each plugin file 
+* If the plugin installation has external dependencies, plugin publisher MUST include them in the plugin package
+* License file is optional but it's recommended to be included in the plugin package
```

Comment:
```suggestion
* License file is optional but it's recommended to be included in the release package
```

Reply:
I find the term `archive` is much more accurate than release/plugin package. See [Grafana doc](https://grafana.com/docs/grafana/latest/administration/plugin-management/#install-a-packaged-plugin) as an example.

Would it be better to use `archive` as the term here?